bin/localstack/wait-local-s3-exists

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+echo "Please make sure the local s3 bucket exists (run './bin/start-local-s3' from another terminal)"
```

Comment:
Question (if-minor): Can we include the start-local-s3 command here? What is the benefit of having a separate terminal?